recipes/cbflib/meta.yaml

```
@@ -8,25 +8,40 @@ package:
 source:
   # Temporarily use git repository until we figure out what changes are
   # needed. Then coordinate with @yayahjb to make a new release.
-  git_url: https://github.com/yayahjb/cbflib.git
   # url: https://github.com/yayahjb/cbflib/archive/CBFlib-{{ version }}.tar.gz
   # sha256: b3ed6cbddc7dd9c565ac116a6eaf44a437e88808921a4e458a26e327e3f66f48
+  - git_url: https://github.com/ndevenish/cbflib
+    git_rev: refactor
 
 build:
   number: 0
+  skip: True  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('fortran') }}
+    # - {{ compiler('fortran') }}  # [not win]
+    - python
+    - cmake
+    - swig
+    - ninja  # [not win]
+    - m4     # [not win]
+
   host:
-    - hdf5
-    - zlib
+    - python
+    - hdf5 <1.12
+    - pcre
+    - pip
+
+  run:
+    - python
+    - hdf5 <1.12
+    - pcre
 
 test:
-  commands:
-    - makecbf --help
+  imports:
+    - pycbf
 
```

```suggestion
  commands:
    - pip check
  requires:
    - pip

```
to ensure environment consistency